2020-01-12 20:23:24,364 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 1 hosts
2020-01-12 20:23:24,472 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 1 hosts
2020-01-12 20:27:08,888 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 1 hosts
2020-01-12 20:27:09,011 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 1 hosts
2020-01-12 20:30:32,647 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-12 20:30:32,758 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 20:31:18,897 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-12 20:31:18,902 - nornir.core.task -    ERROR -      start() - Host 'sandXE.cisco': task 'my_task' failed with traceback:
Traceback (most recent call last):
  File "/Users/pmorvay/PycharmProjects/TechClub20/venv/lib/python3.7/site-packages/nornir/core/task.py", line 85, in start
    r = self.task(self, **self.params)
  File "/Users/pmorvay/PycharmProjects/TechClub20/Norn_01.py", line 7, in my_task
    print(task.host + "/n")
TypeError: unsupported operand type(s) for +: 'Host' and 'str'

2020-01-12 20:31:18,902 - nornir.core.task -    ERROR -      start() - Host 'sandXR.cisco': task 'my_task' failed with traceback:
Traceback (most recent call last):
  File "/Users/pmorvay/PycharmProjects/TechClub20/venv/lib/python3.7/site-packages/nornir/core/task.py", line 85, in start
    r = self.task(self, **self.params)
  File "/Users/pmorvay/PycharmProjects/TechClub20/Norn_01.py", line 7, in my_task
    print(task.host + "/n")
TypeError: unsupported operand type(s) for +: 'Host' and 'str'

2020-01-12 20:31:18,902 - nornir.core.task -    ERROR -      start() - Host 'ASR920.cisco': task 'my_task' failed with traceback:
Traceback (most recent call last):
  File "/Users/pmorvay/PycharmProjects/TechClub20/venv/lib/python3.7/site-packages/nornir/core/task.py", line 85, in start
    r = self.task(self, **self.params)
  File "/Users/pmorvay/PycharmProjects/TechClub20/Norn_01.py", line 7, in my_task
    print(task.host + "/n")
TypeError: unsupported operand type(s) for +: 'Host' and 'str'

2020-01-12 20:31:19,006 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 20:32:23,061 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-12 20:32:23,169 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 20:33:07,617 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-12 20:33:07,621 - nornir.core.task -    ERROR -      start() - Host 'ASR920.cisco': task 'my_task' failed with traceback:
Traceback (most recent call last):
  File "/Users/pmorvay/PycharmProjects/TechClub20/venv/lib/python3.7/site-packages/nornir/core/task.py", line 85, in start
    r = self.task(self, **self.params)
  File "/Users/pmorvay/PycharmProjects/TechClub20/Norn_01.py", line 7, in my_task
    print(f"==== {task.host} ==== \n {result.result} \n")
NameError: name 'result' is not defined

2020-01-12 20:33:07,621 - nornir.core.task -    ERROR -      start() - Host 'sandXR.cisco': task 'my_task' failed with traceback:
Traceback (most recent call last):
  File "/Users/pmorvay/PycharmProjects/TechClub20/venv/lib/python3.7/site-packages/nornir/core/task.py", line 85, in start
    r = self.task(self, **self.params)
  File "/Users/pmorvay/PycharmProjects/TechClub20/Norn_01.py", line 7, in my_task
    print(f"==== {task.host} ==== \n {result.result} \n")
NameError: name 'result' is not defined

2020-01-12 20:33:07,622 - nornir.core.task -    ERROR -      start() - Host 'sandXE.cisco': task 'my_task' failed with traceback:
Traceback (most recent call last):
  File "/Users/pmorvay/PycharmProjects/TechClub20/venv/lib/python3.7/site-packages/nornir/core/task.py", line 85, in start
    r = self.task(self, **self.params)
  File "/Users/pmorvay/PycharmProjects/TechClub20/Norn_01.py", line 7, in my_task
    print(f"==== {task.host} ==== \n {result.result} \n")
NameError: name 'result' is not defined

2020-01-12 20:33:07,725 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 20:33:20,305 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-12 20:33:20,313 - nornir.core.task -    ERROR -      start() - Host 'sandXE.cisco': task 'my_task' failed with traceback:
Traceback (most recent call last):
  File "/Users/pmorvay/PycharmProjects/TechClub20/venv/lib/python3.7/site-packages/nornir/core/task.py", line 85, in start
    r = self.task(self, **self.params)
  File "/Users/pmorvay/PycharmProjects/TechClub20/Norn_01.py", line 7, in my_task
    print(f"==== {task.host} ==== \n {result.result} \n")
NameError: name 'result' is not defined

2020-01-12 20:33:20,314 - nornir.core.task -    ERROR -      start() - Host 'sandXR.cisco': task 'my_task' failed with traceback:
Traceback (most recent call last):
  File "/Users/pmorvay/PycharmProjects/TechClub20/venv/lib/python3.7/site-packages/nornir/core/task.py", line 85, in start
    r = self.task(self, **self.params)
  File "/Users/pmorvay/PycharmProjects/TechClub20/Norn_01.py", line 7, in my_task
    print(f"==== {task.host} ==== \n {result.result} \n")
NameError: name 'result' is not defined

2020-01-12 20:33:20,314 - nornir.core.task -    ERROR -      start() - Host 'ASR920.cisco': task 'my_task' failed with traceback:
Traceback (most recent call last):
  File "/Users/pmorvay/PycharmProjects/TechClub20/venv/lib/python3.7/site-packages/nornir/core/task.py", line 85, in start
    r = self.task(self, **self.params)
  File "/Users/pmorvay/PycharmProjects/TechClub20/Norn_01.py", line 7, in my_task
    print(f"==== {task.host} ==== \n {result.result} \n")
NameError: name 'result' is not defined

2020-01-12 20:33:20,416 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 20:33:39,777 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-12 20:33:39,887 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 20:33:48,805 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-12 20:33:48,915 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 20:34:01,376 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-12 20:34:01,484 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 20:45:39,950 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-12 20:47:32,895 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-12 20:47:33,004 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 20:48:42,465 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-12 20:48:42,573 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 20:49:19,625 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-12 20:49:19,736 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 20:50:21,150 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-12 20:50:21,261 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 20:51:32,593 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-12 20:51:32,698 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 21:02:13,648 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-12 21:02:13,759 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 21:03:21,635 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 1 hosts
2020-01-12 21:03:21,744 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 21:07:11,267 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 1 hosts
2020-01-12 21:07:11,375 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 1 hosts
2020-01-12 21:07:11,480 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 21:10:51,415 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show cdp nei', 'show version', 'show runn']} on 1 hosts
2020-01-12 21:10:51,420 - nornir.core.task -    ERROR -      start() - Host 'sandXE.cisco': task 'get_commands' failed with traceback:
Traceback (most recent call last):
  File "/Users/pmorvay/PycharmProjects/TechClub20/venv/lib/python3.7/site-packages/nornir/core/task.py", line 85, in start
    r = self.task(self, **self.params)
  File "/Users/pmorvay/PycharmProjects/TechClub20/Norn_02.py", line 21, in get_commands
    with open(f"outputs/{task.host.name}-{dt}.txt", "w") as output_file:
FileNotFoundError: [Errno 2] No such file or directory: 'outputs/sandXE.cisco-2020-01-12 21:10:51.419422.txt'

2020-01-12 21:10:51,523 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 21:11:13,939 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show cdp nei', 'show version', 'show runn']} on 1 hosts
2020-01-12 21:11:21,501 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 21:13:55,102 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show cdp nei', 'show version']} on 1 hosts
2020-01-12 21:14:02,682 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 21:16:19,030 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show cdp nei', 'show version']} on 1 hosts
2020-01-12 21:16:26,921 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 21:17:47,627 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show cdp nei', 'show version']} on 1 hosts
2020-01-12 21:19:39,836 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show version']} on 1 hosts
2020-01-12 21:20:50,858 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show version']} on 1 hosts
2020-01-12 21:20:57,171 - nornir.core.task -    ERROR -      start() - Host 'sandXE.cisco': task 'get_commands' failed with traceback:
Traceback (most recent call last):
  File "/Users/pmorvay/PycharmProjects/TechClub20/venv/lib/python3.7/site-packages/nornir/core/task.py", line 85, in start
    r = self.task(self, **self.params)
  File "/Users/pmorvay/PycharmProjects/TechClub20/Norn_02C.py", line 24, in get_commands
    print(f"==== {result[0][result][version]} ====\n")
TypeError: 'Result' object is not subscriptable

2020-01-12 21:20:57,195 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 21:21:47,625 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show version']} on 1 hosts
2020-01-12 21:22:51,537 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show version']} on 1 hosts
2020-01-12 21:22:58,147 - nornir.core.task -    ERROR -      start() - Host 'sandXE.cisco': task 'get_commands' failed with traceback:
Traceback (most recent call last):
  File "/Users/pmorvay/PycharmProjects/TechClub20/venv/lib/python3.7/site-packages/nornir/core/task.py", line 85, in start
    r = self.task(self, **self.params)
  File "/Users/pmorvay/PycharmProjects/TechClub20/Norn_02C.py", line 24, in get_commands
    print(f"==== {result[0]['result']['version']} ====\n")
TypeError: 'Result' object is not subscriptable

2020-01-12 21:22:58,200 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 21:29:41,000 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show version']} on 1 hosts
2020-01-12 21:35:57,366 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show version']} on 1 hosts
2020-01-12 21:36:04,005 - nornir.core.task -    ERROR -      start() - Host 'sandXE.cisco': task 'get_commands' failed with traceback:
Traceback (most recent call last):
  File "/Users/pmorvay/PycharmProjects/TechClub20/venv/lib/python3.7/site-packages/nornir/core/task.py", line 85, in start
    r = self.task(self, **self.params)
  File "/Users/pmorvay/PycharmProjects/TechClub20/Norn_02C.py", line 24, in get_commands
    print(f"==== {result['0']['result']} ====\n")
TypeError: list indices must be integers or slices, not str

2020-01-12 21:36:04,028 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 21:36:19,644 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show version']} on 1 hosts
2020-01-12 21:36:25,730 - nornir.core.task -    ERROR -      start() - Host 'sandXE.cisco': task 'get_commands' failed with traceback:
Traceback (most recent call last):
  File "/Users/pmorvay/PycharmProjects/TechClub20/venv/lib/python3.7/site-packages/nornir/core/task.py", line 85, in start
    r = self.task(self, **self.params)
  File "/Users/pmorvay/PycharmProjects/TechClub20/Norn_02C.py", line 24, in get_commands
    print(f"==== {result[0]['result']} ====\n")
TypeError: 'Result' object is not subscriptable

2020-01-12 21:36:25,753 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 22:03:23,965 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show version']} on 1 hosts
2020-01-12 22:03:30,813 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 22:04:05,094 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show version']} on 1 hosts
2020-01-12 22:04:11,160 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 22:05:55,878 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show version']} on 1 hosts
2020-01-12 22:06:02,314 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 22:06:26,230 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show version']} on 1 hosts
2020-01-12 22:06:32,549 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 22:07:44,325 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show version']} on 1 hosts
2020-01-12 22:07:50,695 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 22:08:46,257 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show version']} on 1 hosts
2020-01-12 22:08:52,513 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 22:09:47,398 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show cdp nei', 'show version', 'show ip route']} on 1 hosts
2020-01-12 22:09:55,175 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 22:10:46,574 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show version', 'show ip interface brief']} on 1 hosts
2020-01-12 22:10:53,752 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-12 22:23:05,393 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show version', 'show ip interface brief']} on 1 hosts
2020-01-12 22:24:49,147 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show version', 'show ip interface brief']} on 1 hosts
2020-01-12 22:24:55,813 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-13 22:38:38,645 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show int desc']} on 1 hosts
2020-01-13 22:38:38,651 - nornir.core.task -    ERROR -      start() - Host 'sandXE.cisco': task 'get_commands' failed with traceback:
Traceback (most recent call last):
  File "/Users/pmorvay/PycharmProjects/TechClub20/venv/lib/python3.7/site-packages/nornir/core/task.py", line 85, in start
    r = self.task(self, **self.params)
  File "/Users/pmorvay/PycharmProjects/TechClub20/Norn_03_verify.py", line 20, in get_commands
    result = task.run(task=netmiko_send_command, command_string=command, use_genie=True)
NameError: name 'command' is not defined

2020-01-13 22:38:38,756 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-13 22:40:04,234 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show int desc']} on 1 hosts
2020-01-13 22:40:10,109 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-13 22:41:29,852 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['int loo 123', 'desc created by nornir']} on 1 hosts
2020-01-13 22:41:29,859 - nornir.core.task -    ERROR -      start() - Host 'sandXE.cisco': task 'get_commands' failed with traceback:
Traceback (most recent call last):
  File "/Users/pmorvay/PycharmProjects/TechClub20/venv/lib/python3.7/site-packages/nornir/core/task.py", line 85, in start
    r = self.task(self, **self.params)
  File "/Users/pmorvay/PycharmProjects/TechClub20/Norn_03.py", line 20, in get_commands
    result = task.run(task=netmiko_send_config, config_commands=command)
NameError: name 'command' is not defined

2020-01-13 22:41:29,966 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-13 22:43:12,927 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show int desc']} on 1 hosts
2020-01-13 22:43:19,065 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-13 22:44:49,110 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show int desc']} on 1 hosts
2020-01-13 22:44:54,995 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-13 22:45:23,256 -  nornir.core -     INFO -        run() - Running task 'netmiko_send_config' with args {'config_commands': ['int loo 123', 'description created by nornir']} on 1 hosts
2020-01-13 22:45:31,965 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-13 22:45:53,586 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show int desc']} on 1 hosts
2020-01-13 22:45:59,630 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-13 22:46:37,566 -  nornir.core -     INFO -        run() - Running task 'netmiko_send_config' with args {'config_commands': ['no int loo 123']} on 1 hosts
2020-01-13 22:46:46,160 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-14 08:22:35,530 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-14 08:22:35,644 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-14 08:23:41,542 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-14 08:23:41,648 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-14 08:25:48,484 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-14 08:25:48,594 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-14 08:26:25,356 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-14 09:52:38,051 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show int desc']} on 1 hosts
2020-01-14 09:52:49,099 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-14 10:31:17,803 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-14 10:31:17,934 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-14 10:32:04,670 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-14 10:32:04,780 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-14 10:33:26,821 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-14 10:33:26,929 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-14 10:34:05,896 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-14 10:34:06,008 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-14 10:34:34,501 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-14 10:38:12,645 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 1 hosts
2020-01-14 10:38:12,756 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 1 hosts
2020-01-14 10:38:12,861 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-14 10:41:00,139 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show cdp nei', 'show version', 'show runn']} on 1 hosts
2020-01-14 10:41:07,537 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-14 10:44:29,918 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show cdp nei', 'show version']} on 1 hosts
2020-01-14 10:44:37,875 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-14 10:46:53,335 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show cdp nei', 'show version']} on 1 hosts
2020-01-14 10:47:01,549 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-14 10:47:12,998 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show version']} on 1 hosts
2020-01-14 10:48:33,755 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show version']} on 1 hosts
2020-01-14 10:48:40,183 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-14 17:12:31,698 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show cdp nei', 'show version']} on 1 hosts
2020-01-14 17:12:56,037 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-14 17:13:50,374 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-14 17:13:50,483 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-14 17:14:01,101 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show cdp nei', 'show version']} on 1 hosts
2020-01-14 17:14:09,746 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-14 17:16:12,525 -  nornir.core -     INFO -        run() - Running task 'get_commands' with args {'commands': ['show cdp nei', 'show version']} on 1 hosts
2020-01-14 17:16:44,931 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-01-15 14:53:25,273 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-01-15 14:53:25,380 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
2020-02-25 11:51:11,509 -  nornir.core -     INFO -        run() - Running task 'my_task' with args {} on 3 hosts
2020-02-25 11:51:11,615 -  nornir.core -     INFO -        run() - Running task 'close_connections_task' with args {} on 3 hosts
